### PR TITLE
Topic 5 improvement tab

### PR DESCRIPTION
**Pull request by audstav**
@seanlip @DubeySandeep PTAL

Changes on this site:
https://oppia-user-guide.readthedocs.io/en/latest/index.html

Note that for the 'Suggestions' task, there was a bug identified, so this section will have to be updated when the bug is fixed.

### CODE REVIEW

**Comment by DubeySandeep** [comment]
@audstav, Can you please resolve the merge conflicts in this PR? (You can find "[how to resolve merge conflicts over](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#address-review-comments-until-all-reviewers-give-lgtm-looks-good-to-me)".)

I see lots of duplicate images as well in this PR, not sure why can you please check?

> Note that for the 'Suggestions' task, there was a bug identified, so this section will have to be updated when the bug is fixed.

Can you please add a link to the bug? :)

**Comment by audstav** [comment]
@DubeySandeep Regarding the duplicate images, I don't see any duplicates? There were 26 new ones added in this PR which is what I see in the 'Files changed' tab.

Here is the fix Brian sent out - [#7719](https://github.com/oppia/oppia/pull/7719)

**Review by brianrodri — CHANGES_REQUESTED** [review]
This looks fantastic! Please also add notes explaining that creators can revisit the tasks they resolve by unticking the "Only Show Open Tasks" checkbox at the top (playthroughs are exceptional however, they get permanently deleted).

File: docs/lesson/improvements.rst
Comment (brianrodri) [inline docs/lesson/improvements.rst]: missing space: s/playthrough/play through

File: docs/lesson/improvements.rst
Comment (brianrodri) [inline docs/lesson/improvements.rst]: Let's say "hidden" instead, remove implies (to me) that they are gone forever. Resolved tasks can be viewed again by unticking the "only show open tasks" checkbox at the top of the improvements tab page.

File: docs/lesson/improvements.rst
Comment (brianrodri) [inline docs/lesson/improvements.rst]: "to take action on this suggestion"

File: docs/lesson/improvements.rst
Comment (brianrodri) [inline docs/lesson/improvements.rst]: Please add comma: "The learner selected the right answer, but explained they..."

**Review by brianrodri — CHANGES_REQUESTED** [review]

**Comment by DubeySandeep** [comment]
> Regarding the duplicate images, I don't see any duplicates? There were 26 new ones added in this PR which is what I see in the 'Files changed' tab.

I think before you resolved merge conflicts it was showing duplicate images from your last PR, now It's fine! :)

**Review by DubeySandeep — COMMENTED** [review]
@audstav, This looks great to me, I'm waiting for @anubhavsinha98's review once he approves I'll go ahead and merge this PR!

(In the meantime you can address @brianrodri's review comment!)

File: docs/lesson/improvements.rst
Comment (audstav) [reply docs/lesson/improvements.rst]: Done

File: docs/lesson/improvements.rst
Comment (audstav) [reply docs/lesson/improvements.rst]: Done

File: docs/lesson/improvements.rst
Comment (audstav) [reply docs/lesson/improvements.rst]: Done

File: docs/lesson/improvements.rst
Comment (audstav) [reply docs/lesson/improvements.rst]: Done

**Comment by audstav** [comment]
@DubeySandeep 
I've incorporated suggested changes and updated the video as well.

**Comment by DubeySandeep** [comment]
@audstav Thanks for the changes, @brianrodri can you please take a final look?

File: docs/lesson/improvements.rst
Comment (anubhavsinha98) [inline docs/lesson/improvements.rst]: To delete any **Answer details** received, checkmark the **Select** box and click on......
You can also delete multiple answer details at a time.

**Review by anubhavsinha98 — CHANGES_REQUESTED** [review]
Hi @audstav, looks good. Just one minor change!
Thanks :)

File: docs/lesson/improvements.rst
Comment (audstav) [reply docs/lesson/improvements.rst]: Done

**Comment by audstav** [comment]
@DubeySandeep Anubhav's changes now incorporated!

**Review by anubhavsinha98 — APPROVED** [review]
LGTM :)

**Review by DubeySandeep — APPROVED** [review]

**Comment by DubeySandeep** [comment]
@brianrodri, Can you please take a final look at the changes made as per your comments?

**Comment by DubeySandeep** [comment]
I'm going to merge this PR as per our email thread and @brianrodri can review the PR once he is back and @audstav can create a follow-up PR if required!

**Pull request merged**
